modules/acm/config/install/acm.connector.yml

```diff
@@ -8,3 +8,5 @@ product_publish: false
 api_version: v2
 filter_root_category: false
 text_format: rich_text
+delete_disabled_skus: true
```

Comment:
missing config fields and schema was failing tests, haven't added update hook specifically for this one as I'm setting default values here, let me know if update hook is required (note: they are already editable via admin form)